File: PHP/UberX.php
<?php

require_once('Car.php');

class UberX extends Car
{
    // Atributes

    private $brand;
    private $model;

    //Constructor

    public function __construct($license, $driver, $brand, $model)
    {
        parent::__construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }

    // Getters & Setters

    public function getBrand()
    {
        return $this->brand;
    }

    public function setBrand($brand)
    {
        $this->brand = $brand;
    }

    public function getModel()
    {
        return $this->model;
    }

    public function setModel($model)
    {
        $this->model = $model;
    }

    //Methods

    public function printDataCar()
    {
        parent::printDataCar();
        echo " Marca : " . $this->brand .
             " Model : " . $this->model;
    }
}

?>

File: Java/UberX.java
package Java;


public class UberX extends Car
{
    // Atributes

    private String brand;
    private String model;
    
    // Constructor

    public UberX(String license, Account driver, String brand, String model)
    {
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }
    
    // Getters & Setters

    public String getBrand() {
        return brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }

    //Methods 

    @Override
    public void setPassengers(Integer passengers) 
    {
        if(passengers == 4)
        {
        super.setPassengers(passengers);
        }else
        {
            System.out.println("Esta categoria solo acepta 4 pasajeros");
        }
    }

    
}
